scripts/release-tools/README.md

```diff
@@ -1,3 +1,7 @@
+# DEPRECATION NOTICE
+
+:warning: The scripts are deprecated and will be removed on October 10, 2022. The functionality is now implemented by GitHub Actions.
```

Comment:
10/10 looks pretty soonish. I have nothing against deleting them on that date, but I would propose the following steps to handle this:
- Unlist those scripts from the release manuals: upstream, downstream, tracker. Just remove all texts that mention them. This can happen on 10.10 👍 
- Delete the `release-tools` folder from the repo after 3.73*

*) I think that is the latest date, we can as well do it earlier.

Reply:
I will invite you to a reminder on 10.10.